src/components/Popover/src/PopoverBubble.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div
+		:class="$s.PopoverBubble"
```

Comment:
I know this was carry over, but on second look "Bubble" is so much of a metaphor of its visual style. Something more generic like "PopoverContainer" may provide a bit more meaning and clarity.